ci/check-stack-project.nix

```diff
@@ -24,7 +24,7 @@ writeScript "check-stack-project.sh" ''
   git add -A nix
 
   # Check if there are changes staged for commit.
-  if git diff-index --cached --quiet HEAD --; then
+  if git diff-index --ignore-all-space --cached --quiet HEAD --; then
```

Comment:
Perhaps the `ignore-space-at-eol` option would be more specific here?

Reply:
It might be, but since whitespace doesn't matter to nix, I go with the simpler `git diff-index -w` option.